src/kubernetes_multus.py

```diff
@@ -0,0 +1,276 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Charm Library used to leverage the Multus Kubernetes CNI in charms.
```

Comment:
Shouldn't this be in lib/?

Reply:
Well it should probably be in a lib for a dummy charm for management of k8s resources.

Reply:
Done